src/sentry/mail/notifications.py

```diff
@@ -51,7 +51,7 @@ def get_subject_with_prefix(
 ) -> bytes:
     prefix = ""
     if isinstance(notification, ProjectNotification):
-        prefix = build_subject_prefix(notification.project)
+        prefix = f"{build_subject_prefix(notification.project)} "
```

Comment:
Just to be a bit overcautious, let's do `f"{build_subject_prefix(notification.project).strip()} "`. I have a feeling some self-hosted users may have already changed this option to be something custom with a space at the end, and we don't want to end up adding a double space here.

Reply:
Maybe `rstrip()`, instead. Who knows if our users have a legit use-case for leading spaces.